test/integ/test_connection.py

```diff
@@ -1192,3 +1192,33 @@ def check_packages(message: str, expected_packages: list[str]) -> bool:
     ) as conn, capture_sf_telemetry.patch_connection(conn, False) as telemetry_test:
         conn._log_telemetry_imported_packages()
         assert len(telemetry_test.records) == 0
+
+
+@pytest.mark.skipolddriver
```

Comment:
Should we have an e2e test? (create a temp config file, read from it and connect)